test/vaadin-list-mixin.html

```diff
@@ -635,13 +637,31 @@
           expect(getComputedStyle(list.items[1]).getPropertyValue('display')).to.equal('none');
           expect(list.items[2].focused).to.be.true;
         });
+
         it('should move focus to next not hidden element on "arrow-up"', () => {
           expect(list.items[0].focused).to.be.true;
           expect(getComputedStyle(list.items[0]).getPropertyValue('display')).to.equal('block');
           arrowUp(list);
           expect(getComputedStyle(list.items[list.items.length - 1]).getPropertyValue('display')).to.equal('none');
           expect(list.items[list.items.length - 2].focused).to.be.true;
         });
+
+        it('should not set tabIndex=0 to hidden items, but the next one in the loop', () => {
+          list._focus(1);
+          [-1, -1, 0].forEach((val, idx) => expect(list.items[idx].tabIndex).to.be.equal(val));
+        });
+
+        it('should skip hidden items to key search', () => {
+          keyDownChar(list, 'b');
+          keyDownChar(list, 'b');
+          expect(list.items[6].focused).to.be.true;
```

Comment:
Rather than using magic numbers like 6 or 4 in the tests, prefer something that is more tolerant to changes in the fixture. Like `expect(list.items.filter(item => item.focused)[0].hidden).to.be.false;`

Reply:
fixed

Reply:
You only fixed the other test. The test below still uses a magic number.

Reply:
Discussed in person. Better to still test that the expected item is focused: `expect(list.items.filter(item => item.textContent === 'Pub')[0].focused).to.be.true;`

Reply:
fixed!